src/Microsoft.AspNetCore.SystemWebAdapters.FrameworkServices/ProxyHeaderModule.cs

```diff
@@ -103,6 +114,31 @@ private static void UseForwardedFor(NameValueCollection requestHeaders, NameValu
         }
     }
 
+    private class ReflectionHelper
+    {
+        private readonly FieldInfo _wrField;
+        private readonly Type _iis7WorkerRequestType;
```

Comment:
nit: is this actually needed? isn't it just to find the second field?

Reply:
Good point, will remove